### DISCUSSION TITLE
Hide Alert "The 'task' is already active"

### DISCUSSION THREAD

**Discussion by marcocaldera** [sweetpad-dev/sweetpad#77] [Any discussions (ideas, questions, ...)]
When I make a change to the app I press the command: "Build & Run" but since the previous build is still running I always need to press "Terminate Task" on the alert that appears.

This requires two button presses for each build while I would like to just press the command once and see the new build appear.

**Comment by hyzyla** [comment]
Hello! Thanks for the suggestion! I've tried to fix that annoying alert in the latest v0.1.46. Now, whenever you run a build task using the buttons on the Sweetpad side panel, it will terminate the previous task and start a new one
